CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## 0.2.8-dev0
+## 0.2.8-dev2
```

Comment:
Do we want to do a release with this PR?

Reply:
Sure, I'll update.